cmake/SuiteSparse-config-install.cmake.in

```diff
@@ -0,0 +1,36 @@
+# Compute locations from <prefix>/@{LIBRARY_DIR@/cmake/suitesparse-<v>/<self>.cmake
+get_filename_component(_SuiteSparse_SELF_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+get_filename_component(_SuiteSparse_PREFIX "${_SuiteSparse_SELF_DIR}" PATH)
+get_filename_component(_SuiteSparse_PREFIX "${_SuiteSparse_PREFIX}" PATH)
+get_filename_component(_SuiteSparse_PREFIX "${_SuiteSparse_PREFIX}" PATH)
+
+# Load the LAPACK package with which we were built.
+find_package(LAPACK CONFIG)
+
+# Load targets from the install tree.
+if(NOT TARGET SuiteSparse::suitesparseconfig)
+  include(${_SuiteSparse_SELF_DIR}/SuiteSparse-targets.cmake)
+endif()
+
+# Report SuiteSparse header search locations.
+set(SuiteSparse_INCLUDE_DIRS ${_SuiteSparse_PREFIX}/include)
```

Comment:
This line is in for backwards compatibility. I think it does not hurt to leave it in